dangerfile.js

```diff
@@ -0,0 +1,73 @@
+const PR_NUMBER = danger.github.pr.number;
+const PR_URL = danger.github.pr.html_url;
+const PR_LINK = `[#${PR_NUMBER}](${PR_URL})`;
+
+function getCleanTitle() {
+  const title = danger.github.pr.title;
+  return title.split(": ").slice(-1)[0].trim().replace(/\.+$/, "");
+}
+
+function getChangelogDetails() {
+  return `
+<details>
+<summary><b>Instructions and example for changelog</b></summary>
+
+Please add an entry to \`CHANGELOG.md\` to the "Unreleased" section under the following heading:
+ 1. **Feat**: For new user-visible functionality.
+ 2. **Fix**: For user-visible bug fixes.
+ 3. **Ref**: For features, refactory and bug fixes in internal operation.
```

Comment:
```suggestion
 1. **feat**: For new user-visible functionality.
 2. **fix**: For user-visible bug fixes.
 3. **ref**: For features, refactors and bug fixes in internal operations.
```

Reply:
sure, thanks for pointing that out @jennmueng.